story.txt

```diff
@@ -1,6 +1,5 @@
-Once upon a time...
-
-Hiroki asked me a question about how to open previous changes.
-
-I completely ignored Hiroki, and decided to buy a fish
-a crsipy fish to slap Ashbin in the hamstrangs.
+Long long ago,
+in a galaxy far far away,
+there was a young boy who played with technology for fun.
```

Comment:
please change this line to anything else......

Reply:
@hirokisaru I made the changes